src/unicode/unicode.cr

```diff
@@ -1,13 +1,12 @@
-# Provides methods that answer questions about unicode characters,
-# and the `Unicode::CaseOptions` enum for special case conversions
-# like Turkic.
+# The `Unicode` module provides methods that answer questions about unicode characters,
+# and the `Unicode::CaseOptions` enum for special case conversions like Turkic.
 #
-# There's no need to use the methods defined in this module
+# There's no need to use the methods defined here
```

Comment:
I think this line here is incorrect. It says "there's no need to use the methods in this module"... but all the methods are `:nodoc:`! So which methods is this comment talking about?

I'd simply hide this comment and say that the `Unicode` module provides just the `Unicode::CaseOptions` enum.

Reply:
@r00ster91 the comment above still applies. It's not a priority but in order to merge this PR, those need to be addressed.

Reply:
Oh, I must've missed this one.

I left this in because this comment will not show up in the API docs. This whole comment will only be visible to people visiting this file on GitHub. And for those these methods are visible. However it doesn't make sense to say that these nodoc methods shouldn't be used only here in this file. So I will remove this section of this comment and add a public comment saying that it provides the `CaseOptions` enum.